src/content/reference/cli.md

```diff
@@ -265,9 +265,14 @@ $ particle subscribe eventName CoreName
 $ particle subscribe eventName 0123456789ABCDEFGHI
 ```
 
+## particle publish
 
-
-
+  Publishes events to the cloud as if they came from the Particle Core/Photon via Spark.publish (deprecated) or Particle.publish.
```

Comment:
when an event comes from the api / various services instead of a device, the coreid / deviceid is kinda random.  In this case it's an identifier used by the api, but I wouldn't count on it to stay that.  Long-term we'll change that to something that makes more sense :)